Documentation/QuickInstall/TheInstallTool/Index.rst

```diff
@@ -22,9 +22,9 @@ The Admin Tool
       Success message after download.
 
 
-#. Create the file FIRST_INSTALL in your web root directory (`public` folder)
-   and reload the page. You will be redirected to the install tool. It will
-   guide you through the steps for installing TYPO3. The Install Tool is
+#. Create an empty file :file:`FIRST_INSTALL` in your web root directory (e.g. `public` folder).
+   Then reload the page. You will be redirected to the installation process. It will
+   guide you through the steps for installing TYPO3. The Admin Tool is
```

Comment:
```suggestion
   guide you through the steps for installing TYPO3. The Install Tool is
```

There is no clear consensus how it should be named but from what I can see it's [still called "Install tool" everywhere](https://github.com/TYPO3/TYPO3.CMS/search?q=install+tool). Also this is the term people are used to so if it is changed, this must be done everywhere at the same time.